fontbe/src/stat.rs

```diff
@@ -64,11 +64,18 @@ impl Work<Context, AnyWorkId, Error> for StatWork {
 }
 
 fn make_stat(static_metadata: &StaticMetadata) -> Stat {
-    let reverse_names: HashMap<_, _> = static_metadata
-        .names
-        .iter()
-        .map(|(key, name)| (name.as_str(), key.name_id))
-        .collect();
+    // Prefer the smallest ID if an existing name can be reused
```

Comment:
specify, with reference if possible, that this is to match FontTools

Reply:
I've now documented this on the corresponding tables with references to the table ID use and name creation in fonttools